lamindb_setup/_load_instance.py

```diff
@@ -128,7 +138,12 @@ def load(
                 f"'{user_info['handle']}' (id: '{locked_by}', name:"
                 f" '{user_info['name']}')."
             )
-        raise RuntimeError(lock_msg)
+        raise InstanceLockedException(lock_msg)
+
+    # this is for testing purposes only
+    if _TEST_FAILED_LOAD:
```

Comment:
@falexwolf this is probably a bad idea, but i don't see how else to trigger an error exactly at the right place.

Reply:
It's OK for now!